Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -599,7 +601,9 @@ def will_run(item):
             try:
                 skipping_pytest_runtest_setup(item)
                 return True
-            except Exception:  # intentionally broad
+            except (Warning, Exception, _pytest.outcomes.OutcomeException) as ex:
+                # intentionally broad to avoid events other than system interrupts
+                warnings.warn(f"Test deselected from execution queue due to {ex}")
```

Comment:
Apparently the previously broader exception clause was required, particularly for pytest exception-types including Skip which all extend BaseExceptions. This should now catch everything, record it as a warning, but not ignore system interrupts.